tests/x11/x3270_ssl.pm

```diff
@@ -32,6 +32,11 @@ sub run {
 
     select_console 'root-console';
 
+    # On s390x platform, make sure that non-root user has
+    # permissions for $serialdev to get openQA work properly.
+    # Please refer to bsc#1195620
+    ensure_serialdev_permissions if (is_s390x);
```

Comment:
If this is a workaround for bsc#1195620, please add record_soft_failure.

Reply:
@STARRY-S 
Please hold on this PR, since I need double check with @Amrysliu, if it is fixed already, we don't need add the comments any more.

However, you can continue use this changes. since it makes sense we use this function for non-root user.

good job!